core/src/main/java/com/adobe/aem/commons/assetshare/content/properties/AbstractComputedProperty.java

```diff
@@ -59,23 +61,40 @@ protected ValueMap getMetadataProperties(Asset asset) {
         return metadataResource.getValueMap();
     }
 
+    protected ValueMap getJcrContentProperties(final Asset asset) {
```

Comment:
I don't think this method is necessary. If you need to pull the `jcr:title` off the `jcr:content` node you can:

```
ValueMap vm = getAssetProperties(asset);
vm.get("jcr:content/jcr:title", String.class);
```